pkg/client/client_test.go

```
@@ -212,23 +221,20 @@ func TestTableStructure(t *testing.T) {
 		Pass:   password,
 		Host:   host,
 		Port:   port,
-		DBName: name,
 		Schema: schema,
 		SSL:    "disable",
 		Limit:  100,
 	}
 
 	c, _ := New(opts)
```

You shouldn't ignore the error here